### DISCUSSION TITLE
[Vue] Newly added objects cannot be modified using controls

### DISCUSSION THREAD

**Discussion by xorb** [fabricjs/fabric.js#9120] [Q&A]
Example video:

[screen-capture.webm](https://github.com/fabricjs/fabric.js/assets/39509710/ba169b3c-11be-4f31-ac48-e363176bfe81)

Demo link:

https://codesandbox.io/p/sandbox/fabric-vue-vr8242

**Comment by xorb** [comment]
Seems like this  issue is specific to vue. It happens when canvas instance is stored inside `reactive/ref` from vue.

If you store it in a regular variable, it works: https://codesandbox.io/p/sandbox/fabric-vue-forked-p6389x

If someone knows why it happens, please reply.

Regards,

**Reply by geminigeek** [reply]
try like this 
```
canvas.value.add(markRaw(rect))
```
